rpi3.mk

```diff
@@ -44,11 +47,10 @@ ifeq ($(CFG_TEE_BENCHMARK),y)
 all: benchmark-app
```

Comment:
All the benchmark-app stuff can be removed too.

Reply:
Right, but when doing that I start to wonder how do we propagate the `CFG_TEE_BENCHMARK` flag?

Reply:
In other words, shouldn't the CFG... be moved to surround this?
https://github.com/OP-TEE/build/blob/master/common.mk#L218-L219

Reply:
Never mind ... spotted it now further down 2 seconds after posting the comments :)

Reply:
https://github.com/OP-TEE/build/blob/d496f98386525b0379116707de56883057469e8d/common.mk#L223-L225